test/services/forest_liana/scope_validator_test.rb

```diff
@@ -0,0 +1,174 @@
+module ForestLiana
+  class ScopeValidatorTest < ActiveSupport::TestCase
+    test 'Request with aggregated condition filters should be allowed if it matches the scope exactly' do
+      allowed = ForestLiana::ScopeValidator.new({
+          'aggregator' => 'and',
+          'conditions' => [
+            { 'field' => 'name', 'value' => 'john', 'operator' => 'equal' },
+            { 'field' => 'price', 'value' => '2500', 'operator' => 'equal' }
+          ]
+        }, []
+        ).is_scope_in_request?({
+          user_id: '1',
+          filters: JSON.generate({
+            aggregator: 'and',
+            conditions: [
+              { field: 'name', value: 'john', operator: 'equal' },
+              { field: 'price', value: '2500', operator: 'equal' }              
+            ]
+          })
+        })
```

Comment:
[MISC] Out of readability concerns, could you split the ScopeValidator instance creation and the call to `is_scope_in_request?` ?